Gale Databases.js

```diff
@@ -1,190 +1,211 @@
+	return GaleZotero.process(doc);
 }
+
 /** BEGIN TEST CASES **/
+// eslint-disable-next-line no-unused-vars
 var testCases = [
 	{
-		"type": "web",
-		"url": "http://go.galegroup.com/ps/i.do?p=PROF&u=nysl_ce_syr&id=GALE|A213083272&v=2.1&it=r&sid=PROF&asid=a8973dd8",
-		"items": [
+		type: "web",
+		url: "https://go.qa.galetesting.com/ps/retrieve.do?tabID=T002&resultListType=RESULT_LIST&searchResultsType=SingleTab&searchType=BasicSearchForm&currentPosition=1&docId=GALE%7CA598621601&docType=Critical+essay&sort=Relevance&contentSegment=ZONE-Exclude-FT&prodId=AONE&contentSet=GALE%7CA598621601&searchId=R1&userGroupName=zotero&inPS=true",
+		items: [
 			{
-				"itemType": "magazineArticle",
-				"title": "Improving a counselor education Web site through usability testing: the bibliotherapy education project",
-				"creators": [
-					{
-						"lastName": "McMillen",
-						"firstName": "Paula S.",
-						"creatorType": "author"
-					},
+				itemType: "magazineArticle",
+				title: "\"Real\" Mind Style and Authenticity Effects in Fiction: Represented Experiences of War in Atonement",
+				creators: [
 					{
-						"lastName": "Pehrsson",
-						"firstName": "Dale-Elizabeth",
-						"creatorType": "author"
+						lastName: "Nuttall",
+						firstName: "Louise",
+						creatorType: "author"
 					}
 				],
-				"date": "December 2009",
-				"ISSN": "0011-0035",
-				"archive": "Educators Reference Complete",
-				"issue": "2",
-				"language": "English",
-				"libraryCatalog": "Gale",
-				"pages": "122-",
-				"publicationTitle": "Counselor Education and Supervision",
-				"shortTitle": "Improving a counselor education Web site through usability testing",
-				"url": "http://link.galegroup.com/apps/doc/A213083272/PROF?u=nysl_ce_syr&sid=PROF&xid=a8973dd8",
-				"volume": "49",
-				"attachments": [
+				date: "2019",
+				ISSN: "0039-4238",
+				archive: "Gale Academic OneFile",
+				issue: "2",
+				language: "English",
+				libraryCatalog: "Gale",
+				pages: "215-",
+				publicationTitle: "Style",
+				shortTitle: "\"Real\" Mind Style and Authenticity Effects in Fiction",
+				url: "https://link.qa.galetesting.com/apps/doc/A598621601/AONE?u=zotero&sid=zotero&xid=c1dc0ef6",
+				volume: "53",
+				attachments: [
 					{
-						"title": "Snapshot"
+						title: "Snapshot"
 					}
 				],
-				"tags": [
+				tags: [
+					{
+						tag: "Atonement (McEwan, Ian) (Novel)"
+					},
 					{
-						"tag": "Bibliotherapy"
+						tag: "Authenticity"
 					},
 					{
-						"tag": "Counseling"
+						tag: "English writers"
 					},
 					{
-						"tag": "Counselling"
+						tag: "Literary styles"
 					},
 					{
-						"tag": "Usability testing"
+						tag: "McEwan, Ian"
 					},
 					{
-						"tag": "Web sites (World Wide Web)"
+						tag: "War stories"
 					}
 				],
-				"notes": [],
-				"seeAlso": []
+				notes: [
+					{
+						note: "<p>Northern Illinois University</p>"
+					}
+				],
+				seeAlso: []
```

Comment:
Please revert all the quote changes in these test cases. These tests should be generated by Scaffold and shouldn't be reformatted.